assets/sass/06-components/posts-and-pages.scss

```diff
@@ -2,15 +2,21 @@
 	// This class is required to pass ThemeCheck.
 }
 
-.sticky-post {
```

Comment:
I removed this for now because it is not used by the theme. It is a left over from the "featured" label that is used in Seedlet. Can be reconsidered for  https://github.com/WordPress/twentytwentyone/issues/532